es-vue-base/src/lib-components/EsHorizontalList.vue

```diff
@@ -24,7 +24,14 @@
         </div>
         <div
             v-if="dots"
-            class="dots-holder list-unstyled w-100 mt-100 text-center">
+            class="dots-holder list-unstyled w-100 mt-100 text-center d-flex align-items-center justify-content-center">
+            <es-button
+                v-if="arrows"
+                variant="link"
+                class="text-black d-none d-lg-block p-0 mr-200"
+                @click="arrowClick('left')">
+                <icon-chevron-left />
+            </es-button>
```

Comment:
Can we throw a `<span class="sr-only">Previous</span>` in here for accessibility purposes, so the function if this button is clear to screen readers?